openjdk/build.gradle

```diff
@@ -268,6 +268,22 @@ def addNativeTest(nativeClassifier) {
             executable = javaExecutable
             testClassesDir = testSourceSet.output.classesDir
 
+            // TODO(nmittler): Is there a way to copy all properties of the test task?
```

Comment:
Yeah, I think .properties is automatic via the same mechanism that enables .foo references on objects.